services/web/client/source/class/qxapp/desktop/StudyEditor.js

```diff
@@ -239,14 +255,6 @@ qx.Class.define("qxapp.desktop.StudyEditor", {
           this.__nodeView.buildLayout();
           if (node.isInKey("file-picker")) {
             widget = new qxapp.file.FilePicker(node, this.getStudy().getUuid());
```

Comment:
Is this part ever reached?

Reply:
you mean the part where the filepicker is assigned to widget, or the else ``widget = this.__nodeView``
Both parts should be reached, when it is not the root node and the node is not a container node, that is most of the times, right?

Reply:
My bad I missed up line numbers